app/src/main/java/ch/epfl/sdp/map/MapsActivity.java

```diff
@@ -5,20 +5,25 @@
 import android.os.Bundle;
 import android.widget.Button;
 
-import androidx.fragment.app.FragmentActivity;
-
 import com.google.android.gms.maps.GoogleMap;
 import com.google.android.gms.maps.OnMapReadyCallback;
 import com.google.android.gms.maps.SupportMapFragment;
 
+import androidx.fragment.app.FragmentActivity;
 import ch.epfl.sdp.R;
+import ch.epfl.sdp.database.FirestoreUserData;
+import ch.epfl.sdp.database.UserDataController;
+import ch.epfl.sdp.entity.Player;
 import ch.epfl.sdp.game.Game;
 import ch.epfl.sdp.item.Healthpack;
 import ch.epfl.sdp.item.Item;
 import ch.epfl.sdp.item.Scan;
 
 public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+    public static final String TAG = "Test Firebase";
     public static final MapApi mapApi = new GoogleMapApi();
+    public static UserDataController userDataController = new FirestoreUserData();
+    Player currentUser = new Player(7.9592, 47.0407, 22, "startGame2", "startGame2@gmail.com");

```

Comment:
Do we now have the current user? We could replace this mock user

Reply:
Yes, but  the part of the code to get the currentUser is still in startGame branch, not in master branch, so we hardcoded one here.

Reply:
Ok :)